db-migrations/000005_adding_owners.up.sql

```diff
@@ -0,0 +1,22 @@
+-- schema changes for account owners and champions
+BEGIN;
+
+create table person(
+    id serial primary key,
+    login varchar not null,
+    email varchar not null,
+    name varchar not null,
+    valid boolean
```

Comment:
debating if this should be null or even including, the idea of a person being `valid` is a bit odd